### ISSUE TITLE
DM application command - command option type user

### ISSUE THREAD

**Issue by HeroClay** [Breq16/flask-discord-interactions#62]
A application command with an user command option type raises a _KeyError_ when used in a dm
Like this:
```
@discord.command(name="pick", description="Pick a user", options=[{
        "name": "user",
        "description": "The user to pick",
        "type": CommandOptionType.USER,
        "required": True
    }])
```
The Error:
```
[2021-12-16 19:14:37,564] ERROR in app: Exception on /interactions [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/heroclay/.local/lib/python3.8/site-packages/flask_discord_interactions/discord.py", line 514, in interactions
    return jsonify(self.run_command(request.json).dump())
  File "/home/heroclay/.local/lib/python3.8/site-packages/flask_discord_interactions/discord.py", line 434, in run_command
    return command.make_context_and_run(self, current_app, data)
  File "/home/heroclay/.local/lib/python3.8/site-packages/flask_discord_interactions/command.py", line 175, in make_context_and_run
    args, kwargs = context.create_args()
  File "/home/heroclay/.local/lib/python3.8/site-packages/flask_discord_interactions/context.py", line 178, in create_args
    return self.create_args_chat_input()
  File "/home/heroclay/.local/lib/python3.8/site-packages/flask_discord_interactions/context.py", line 228, in create_args_chat_input
    return create_args_recursive({"options": self.options}, self.resolved)
  File "/home/heroclay/.local/lib/python3.8/site-packages/flask_discord_interactions/context.py", line 210, in create_args_recursive
    member_data = resolved["members"][option["value"]]
KeyError: 'members'
```

**Comment by Breq16** [comment]
Thanks for the bug report!

When you say "in a DM," do you mean a direct message between you and the bot? Most people using this library don't set up a "bot user," and purely webhook-based bots can't be DM'd. That doesn't mean I won't support this use case--I'm just trying to understand how to reproduce the issue you describe.

**Comment by HeroClay** [comment]
Thanks for replying! Yes, i meant the direct message between the bot and me. In it, any other slash command with CommandOptionType.STRING works perfectly fine, I can even check with `if ctx.author.display_name: ...` whether its a dm channel or not. But with the CommandOptionType.USER, it just gives back an error.

**Comment by Breq16** [comment]
Cool, thanks for the info. I'm able to reproduce this issue on my machine.

Even when the commands work, it looks like a lot of information from the `Context` object is missing inside of a DM. I'll try and fix the breaking issues first, then document some of these differences somewhere.

Thanks for bringing this to my attention! I'll try to push out a release with a fix over the next few days.

**Comment by Breq16** [comment]
Should be fixed in v1.2.0. Upgrade with

```
pip3 install Flask-Discord-Interactions --upgrade
```

and feel free to reopen if you still experience issues!

**Comment by HeroClay** [comment]
Thanks!
